tezos-release.nix

```diff
@@ -1,36 +0,0 @@
-# SPDX-FileCopyrightText: 2021 Oxhead Alpha
-# SPDX-License-Identifier: LicenseRef-MIT-OA
-
-{ writeTextDir, runCommand, buildEnv, binaries, arm-binaries, commonMeta, replaceStrings, protocols }:
-let
-  release-binaries = import ./nix/build/release-binaries.nix protocols;
-  version = replaceStrings ["refs/tags/"] [""] commonMeta.branchName;
-  release-notes = writeTextDir "release-notes.md" ''
```

Comment:
I don't think that this only took care of signing packages. Moreover AFAIK there is some duplication built in to allow people to choose between `nix` or `docker` as they prefer.

Reply:
it's not related to PR, removed by mistake